### DISCUSSION TITLE
Heat conduction via different materials (with different heat properties) through a solid - FDS

### DISCUSSION THREAD

**Discussion by Ryadh20** [firemodels/fds#12944] [Q&A]
Hello everyone, 
I was wondering how good FDS is doing heat trasnfers via conduction through materials. 

In my case : I have two different solids : 

-One is a PVC (very résistant to heat). 
-The second one is made of steel (very conductive) 

Both of my surfaces are set to "Exposed" on pyrosim. and then I export the file to the FDS format to get better results. 

As to the properties of the Steel : 

![image](https://github.com/firemodels/fds/assets/169986006/4f4af11e-d3c7-4928-b680-f14bec897944)

For the boundaries created for the PVC : 

![image](https://github.com/firemodels/fds/assets/169986006/6edbaa39-8ed7-450f-9aac-4331de795798)

As to the surfaces, they both have the same dimensions and properties : 

![image](https://github.com/firemodels/fds/assets/169986006/87746b8b-69b2-4718-81a7-4af053857895)

this is what the smv results look like :

![image (2)](https://github.com/firemodels/fds/assets/169986006/df22b31b-fd44-4994-9753-152eec43ae44)

The slices I used : 

![image](https://github.com/firemodels/fds/assets/169986006/91204f93-8d22-4910-9007-1abee5f66b90)

I find it very weird, that with different radiation coefficient and conduction coefficient, and at the same instant t, the température is the same on the same point in the Temp Slices 

I hope my request is clear for everyone,

Thanks for your help in advance,

Ryadh

**Comment by drjfloyd** [comment]
The large, 600 C, red block is heating the gas. You are viewing a slice file of gas temperature. The two different plates are probably just not having a significant impact on the temperature of the gas where the primary source of heat is the 600 C hot block. Add an output for the wall temperature so you can see the temperature of the plates themselves.

In general it is better to attach the .fds file (change the extension to .txt in order to attach it) than just give screen shots. That way others can run the full input file if needed.

**Reply by Ryadh20** [reply]
Hello. Thanks for your reply. 

Actually, I am not interested in the température of the two plates. more like the temperature that the gray plate is taking (which is unchanged for both cases knowing that the black plate has two materials for two different cases - which is supposed to give different temperatures that the gray plate should get... 

I don't know if this makes it clearer ? 




Here are the attached files : 
[Heat_transfert_model_-_PVC.txt](https://github.com/firemodels/fds/files/15338951/Heat_transfert_model_-_PVC.txt)
[Heat_transfert_model_-_Steel.txt](https://github.com/firemodels/fds/files/15338953/Heat_transfert_model_-_Steel.txt)


Thanks, 



Ryadh

**Comment by drjfloyd** [comment]
You have no outputs that are measuring the temperatures of the solid surfaces. All your outputs are gas phase quantities.  

Check the units on your specific heat inputs. Your steel specific heat is an order of magnitude higher than typical values for steel.

**Reply by Ryadh20** [reply]
Yes, but again, I am only interested in the gas phase quantities. We do agree that the gas phase quantities above a solid that is supposed to dissipate heat, should be less valuable than in a case where a solid is not heat-isolating at all ? 

And for the steel properties, I know, I did that deliberately, to make sure FDS will in worst cases, make a difference in calculations for two different materials having such a large gap - but from what I see so far he doesn't mind at all the material properties, which I can't explain.

**Comment by drjfloyd** [comment]
You have access to every quantity being predicted in FDS but are only outputting the gas temperature. This is not enough for you to determine what is happening.  What is the heat transfer from the hot block to the gas? From the hot block to the first plate? From the first plate to the gas? Over 60 s are the plate temperatures different enough given the flow field that develops to impact the gas temperature noticeably over the plates?
